docker/run-common.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-./manage.py syncdb --noinput
+./manage.py migrate --noinput --database bedrock
```

Comment:
I believe this file can be removed altogether, along with `docker/run-dev.sh` which is no longer used.